torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2714,10 +2714,7 @@ def sample_inputs_adaptive_avg_pool1d(op_info, device, dtype, requires_grad, **k
     )
 
     for input_shape, output_size in cases:
-        # Batched
         yield SampleInput(make_arg(input_shape), args=(output_size,))
-        # Unbatched
-        yield SampleInput(make_arg(input_shape[1:]), args=(output_size,))
 
```

Comment:
I think we should remove different cases for batched/unbatched SampleInputs. As fixed in #80891.

Reply:
We do want to pass unbatched input to verify for unbatched support so this needs to stay. (also applies to all cases below).

Reply:
Ooops!! I'll undo these changes.